### DISCUSSION TITLE
Keystone debug! and source map

### DISCUSSION THREAD

**Discussion by pahaz** [keystonejs/keystone#6749] [Questions] [answered]
Hi! Does anyone have an example of how to write unit tests for keystone-based projects? I want to debug with Keystone TypeScript source maps. But it looks like there is no source map inside the NPM package.

**Comment by bladey** [comment, marked as answer]
Hey @pahaz,

We're using GitHub discussions for feature requests only, could you please ask this question in the Keystone Community Slack and we'll assist - https://community.keystonejs.com

Thank you!
